tests/components/xiaomi_miio/test_vacuum.py

```diff
@@ -521,17 +522,23 @@ async def setup_component(hass, entity_name):
     """Set up vacuum component."""
     entity_id = f"{DOMAIN}.{entity_name}"
 
-    await async_setup_component(
-        hass,
-        DOMAIN,
+    hass.config.components.add(XIAOMI_DOMAIN)
+    config_entry = config_entries.ConfigEntry(
```

Comment:
We create the config entry here.

Please use a MockConfigEntry instead.
https://github.com/home-assistant/core/blob/efb172cedd8a54a2a508e27b4110e3b7f6dc7bbd/tests/common.py#L735

Reply:
Thanks for the help, sorry for not getting it earlier.
just implemented it.